src/test/java/cz/cvut/kbss/termit/persistence/dao/changetracking/ChangeRecordDaoTest.java

```diff
@@ -113,11 +126,16 @@ private UpdateChangeRecord generateUpdateRecord(Instant timestamp, URI changedOb
     @Test
     void findAllRetrievesChangeRecordsRelatedToSpecifiedAsset() {
         enableRdfsInference(em);
-        final Term asset = Generator.generateTermWithId();
+        final Term asset = Generator.generateTermWithId(vocabulary.getUri());
+        transactional(() -> {
+            em.persist(vocabulary);
+            em.persist(asset, persistDescriptor(vocabulary.getUri()));
+        });
         final List<AbstractChangeRecord> records = IntStream.range(0, 5).mapToObj(
                 i -> generateUpdateRecord(Instant.ofEpochMilli(System.currentTimeMillis() - i * 10000L),
```

Comment:
Could you replace the `Instance.ofEpochMill` with `Utils.timestamp`? I know it is not a part of this work, but I would like to eventually use that method everywhere so that we are consistent with the way we create timestamps...